diagnostic.md

```diff
@@ -6,76 +6,77 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```bash
-// your response here
+localhost:XXXX/books/4
+Also, via curl request
 ```
 
 In what file does the above code go?
 
-```bash
-// your response here
+```txt
+Not sure what is being looked for here. It could be in a a script file so the curl request could be run from the command line. It could also be the routes file that is like Ruby's air traffic controller.
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
-```bash
-// your response here
+```txt
+This function is like pushing to whatever changes we have made to our code to GitHub. This updates our rails API with the most recent chages made.
 ```
```

Comment:
It's okay to think of it like git, but remember that these are just changes in our back end and we're updating it with the changes we've made in the text editor.